bin/dasht-query-line

```diff
@@ -28,8 +28,8 @@
 #
 # Searches for *PATTERN* in all installed [Dash] docsets, optionally searching
 # only in those whose names match *DOCSET*s, by calling dasht-query-exec(1)
-# and emits the results in groups of lines, as described in "Results" below.
-# However, if no results were found, this program exits with a nonzero status.
+# and emits the results as TSV. However, if no results were found, this program
```

Comment:
Please spell out "TSV" in prose: "as TSV" -> "in Tab Separated Vector (TSV) format"